### DISCUSSION TITLE
how to set cornerSize for all objects in v6(ts)?

### DISCUSSION THREAD

**Discussion by wuxingping** [fabricjs/fabric.js#8640] [General]
how to set cornerSize for  all objects in v6(ts)?

**Comment by asturur** [comment]
hi @wuxingping we are still working on that part.
As of today setting it on the prototype of object should still work:

FabricObject.prototype.cornerSize = number.

if you do that as a configuration before app start.

We are removing prototype fiddling as a standard practice because adopting es classes basically cut ourselves out of it.
We are setting up an object with default values and you will be able to change that.
To change cornerSize dinamically you will need to loop through the objects or alternatively define sizeX and sizeY in your control set

**Reply by wuxingping** [reply]
ok . thanks a lot

**Reply by wuxingping** [reply]
> hi @wuxingping we are still working on that part. As of today setting it on the prototype of object should still work:
> 
> FabricObject.prototype.cornerSize = number.
> 
> if you do that as a configuration before app start.
> 
> We are removing prototype fiddling as a standard practice because adopting es classes basically cut ourselves out of it. We are setting up an object with default values and you will be able to change that. To change cornerSize dinamically you will need to loop through the objects or alternatively define sizeX and sizeY in your control set

how to import FabricObject?i try to do like that: import {FabricObject} from 'fabric'.its wrong.
